api/v1/app.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+"""Checking..."""
+
+from flask import Flask
+from models import storage
+from api.v1.views import app_views
+from os import getenv
+
+
+app = Flask(__name__)
+
+app.register_blueprint(app_views, url_prefix="/api/v1")
+
+@app.teardown_appcontext
```

Comment:
Can you please add description on this method?

Reply:
Of course, I'll do this changes in the next commit.